src/executor/exec_pass.h

```diff
@@ -86,6 +86,9 @@ class OpExecutor {
   virtual OpStatePtr state() const {
     return OpStatePtr();
   }
+  
+ protected:
+  std::vector<NDArray> in_array_fallback;
```

Comment:
Do we really need this as a class data member? Or can we just declare it as a local variable when it is created because  I don't see it used anywhere else in the flow.